lib/jazzy/docset_builder.rb

```diff
@@ -8,20 +8,24 @@ class DocsetBuilder
       include Config::Mixin
 
       attr_reader :output_dir
+      attr_reader :source_dir
       attr_reader :source_module
       attr_reader :docset_dir
       attr_reader :documents_dir
 
       def initialize(output_dir, source_module)
-        @output_dir = output_dir + 'docsets'
         @source_module = source_module
-        @docset_dir = @output_dir + "#{source_module.name}.docset"
+        @docset_dir = output_dir +
+          (config.docset_path || "docsets/#{source_module.name}.docset")
+        @source_dir = output_dir
```

Comment:
this is confusing:

```ruby
@source_dir = output_dir
@output_dir = docset_dir.parent
```

Source is output, but output is docset's parent? Perhaps rename these variables to reflect the intention here?